### DISCUSSION TITLE
HRRRv3 19-36hr access

### DISCUSSION THREAD

**Discussion by AGoldbergNG** [NOAA-GSL/GSL-Regional-Model-Forum#7] [Q&A] [answered]
Hi, I am a data scientist at a large northeast power distribution company and we are building an outage forecast system in part using HRRR as the main weather data input source. We need to be able to forecast out to at least 24 hours, if not the full 48 hours available on v4. 

To build the model, we need a fairly large historical archive of data and since hrrr v4 only has about a year of history, we'd like to use the v3 data. Is there a source of archived hrrr v3 data that includes the 19-36 hourly forecasts? I see pando, google, and aws archives only go out to 18 hours. 

Thanks,
Andrew

**Comment by blaylockbk** [comment, marked as answer]
As far as I understand, HRRRv3 goes out to 36 hours for only the 00, 06, 12, and 18 UTC runs. For all other hours it only goes out to 18 hrs.

**Reply by AGoldbergNG** [reply]
Thank you for the quick response! My understanding is the same, however, it appears that the archives that I've found (pando, aws, google) chose not to include the 19-36 hourly data on those 4 daily runtimes. 

From their faq (https://home.chpc.utah.edu/~u0553130/Brian_Blaylock/hrrr_FAQ.html)
HRRRv3 was implemented at NCEP on July 12, 2018 beginning with the 12z run. Version 3 extends the forecast period for the 00, 06, 12, and 18 run out to 36 hours, but we have decided not to archive the f19-f36 files. Improvements in version 3 include updated Thompson microphysics, improved MYNN PBL scheme, updated land surface model with 15 second MODIS data, refined roughness lengths for certain land use types, and additional improvements in assimilation methods including lightning, radar radial velocity, and TAMDAR.

I've manually checked this, and those files indeed do not appear available from those sources. 

Thanks,
Andrew

**Reply by blaylockbk** [reply]
The 19-36 hr files appear to be on AWS. For example: https://noaa-hrrr-bdp-pds.s3.amazonaws.com/index.html#hrrr.20200823/conus/
![image](https://user-images.githubusercontent.com/6249613/132403113-61fb3090-eec7-4986-9dd7-ff015660e6a6.png)

**Reply by AGoldbergNG** [reply]
These appear to be HRRRv4, which has hours 0-48 in pando. The issue is with HRRRv3, which seems to stop at hour 18 on google and pando. Oddly, my public access on AWS appears to stop at 20160823 (HRRRv2).  (Apologies for late reply, have been on pto)
![image](https://user-images.githubusercontent.com/47151268/133111647-307a1e72-5692-45a1-adc2-2f0ea8ce8497.png)

**Reply by AGoldbergNG** [reply]
Apologies -- upon searching the AWS archive (and not merely fast-forwarding to the end) it appears that it does include the most up-to-date V4 files, and the V3's that I've viewed go out to 24 hours (but not 36). Thanks.

**Reply by blaylockbk** [reply]
Glad you found the files! I have a python package called Herbie that I use to search and download files. If you use python, that might be helpful to you https://github.com/blaylockbk/Herbie

**Comment by brianhempel** [comment]
I have ~2.5yrs of 0Z,6Z,12Z,18Z HREF data (the mean and prob fields) out to f36. You'd have to send me hard drives for it though

Based on my experience with https://twitter.com/nadocast trained model performance would likely be similar to the HRRR

**Reply by AGoldbergNG** [reply]
Providing us access to the HREF archives would be extremely helpful. I've contacted several sources (including Brian Roberts, the modeler), with no one aware of a publicly (or privately) available archive. 

Do you have a sense of how large the dataset is? We should be able to provide the available hard drives assuming the size is manageable.

**Reply by brianhempel** [reply]
Yes, I started my own archive in mid-2018 because I wasn't aware of a public one. I think the biggest data gap is Nov 2020 to mid March 2021 when the first 8TB hard drive filled up before I bothered to connect a second one. So ~12TB is enough; although 2x8TB external HD's might be the cheapest option, or my machines have extra SATA bays that can accept 2.5 or 3.5 inch drives. Email brianhempel@uchicago.edu for more

**Reply by brianhempel** [reply]
Note it's the mean and prob ensemble products for the CONUS only. I don't have the individual members—that would be at least an order of magnitude more data!

For example, here's the [mean](https://github.com/brianhempel/nadocast/blob/master/models/href_mid_2018_forward/common_layers_mean.txt) and the [prob](https://github.com/brianhempel/nadocast/blob/master/models/href_mid_2018_forward/common_layers_prob.txt) layers that are shared throughout that whole time period.

**Comment by cshartsough** [comment]
Hi Andrew and Brian,

It looks like you may have worked out a plan for accessing the HRRRv3 data that you need.  I just wanted to point you to our HRRR web page, which has some links to our archived datasets about halfway down the page.

[https://rapidrefresh.noaa.gov/hrrr/](https://rapidrefresh.noaa.gov/hrrr/
)
Feel free to reach out if there is anything more you need.  I don't have all teh answers, but I know the people who do.  ;-)

Best wishes,
Craig Hartsough
NOAA/GSL
Craig.Hartsough@noaa.gov

**Comment by kirklholub** [comment]
FYI, the official HRRR data archive is at NOAA NCEI:
https://www.ncei.noaa.gov/products/weather-climate-models/rapid-refresh-update

--Kirk

On Tue, Sep 14, 2021 at 12:06 PM Andrew Goldberg ***@***.***>
wrote:

> Apologies -- upon searching the AWS archive (and not merely
> fast-forwarding to the end) it appears that it does include the most
> up-to-date V4 files, and the V3's that I've viewed go out to 24 hours (but
> not 36). Thanks.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-GSL/GSL-Regional-Model-Forum/discussions/7#discussioncomment-1329252>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AGR46I4B7FRIIQNDLJIFTN3UB6FLTANCNFSM5DTB4XZQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 


*************************************************************************************

*
        *

* Kirk L. Holub (NOAA Federal Staff)
        *

* NOAA OAR Global Systems Laboratory
        *

* Information and Technology Services Division
        *

* 325 Broadway                       >  303-748-6390 (cell -- primary at
this time) *

* Bldg 33/2B146/x6642                >  202-709-3967 (office -- expect a
delay)     *

* Boulder, CO  80305-3328            >  303-748-6390 (FAX -- expect a
longer delay) *

*
        *

***********************************   Disclaimer
**********************************

*
        *

*   The contents of this message are mine personally and do not reflect any
      *

*   position of the United States of America (USA), or the US Department of
      *

*   Commerce (DoC), or the National Oceanic and Atmospheric Administration
(NOAA).  *

*
        *

*************************************************************************************

**Reply by AGoldbergNG** [reply]
Thanks -- although that appears to be exclusively for the earlier 'Rapid Refresh' models. Are you aware of an HRRR focused historical archive outside of Brian Blaylock's work? 

The nomad's 'archive' only appears to go back 2 days, although I may not have found the correct link: 
https://nomads.ncep.noaa.gov/pub/data/nccf/com/hrrr/prod/

**Reply by AGoldbergNG** [reply]
Apologies here -- it appears that NOAA manages the AWS / Google Cloud archives. Bit of a learning curve here.
